common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -214,35 +214,40 @@ protected boolean processRedirectUrl(@NonNull final WebView view, @NonNull final
 
     private boolean processWebsiteRequest(@NonNull final WebView view, @NonNull final String url) {
         final String methodName = "#processWebsiteRequest";
-        final PackageHelper packageHelper = new PackageHelper(getActivity().getPackageManager());
-        final Context applicationContext = getActivity().getApplicationContext();
-        if (url.startsWith(AuthenticationConstants.Broker.BROWSER_DEVICE_CA_URL)
-                && packageHelper.isPackageInstalledAndEnabled(applicationContext, AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME)
-                && packageHelper.isPackageInstalledAndEnabled(applicationContext, AuthenticationConstants.Broker.IPPHONE_APP_PACKAGE_NAME)
-                && AuthenticationConstants.Broker.IPPHONE_APP_SIGNATURE.equals(
-                packageHelper.getCurrentSignatureForPackage(AuthenticationConstants.Broker.IPPHONE_APP_PACKAGE_NAME))) {
-            // TODO: This flow should really check if the Microsoft Intune or the Company Portal apps are installed,
-            // which is the correct client app to launch depending on the enrollment, and launch that app, to permanently skip the browser.
-            // Also it must handle 3rd party MDMs as appropriate, which will depend on the browser flow determining the authority.
-            Logger.info(TAG + methodName, "It is a device CA request on IPPhone. Company Portal is installed.");
-            try {
-                Logger.verbose(TAG + methodName, "Sending intent to launch the CompanyPortal.");
-                final Intent intent = new Intent();
-                intent.setComponent(new ComponentName(AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME,
-                        AuthenticationConstants.Broker.COMPANY_PORTAL_APP_LAUNCH_ACTIVITY_NAME));
-                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-                getActivity().startActivity(intent);
-            } catch (final SecurityException ex) {
-                Logger.warn(TAG + methodName, "Failed to launch Company Portal, falling back to browser.");
-                openLinkInBrowser(url);
+
```

Comment:
nit : most of the lines in this method are very long, way beyond 100 chars, see if you can format to improve readability